src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/ChimericAlignmentUnitTest.java

```diff
@@ -7,8 +7,6 @@
 import htsjdk.samtools.SAMRecord;
```

Comment:
This files might have conflicts after https://github.com/broadinstitute/gatk/pull/3475

Reply:
Yep. Thanks for pointing out.
I do plan to go over the SV tests when I have spare time over the weekend, and update them as appropriate.
Maybe I'll even name you as the reviewer/victim :smirk:

Reply:
For the test code I'm happy to review as I did the refactor of the base test for my own benefit; I do not think that I will be useful for the rest of the SV code if you mean that!

Reply:
definitely not the sv code. just those 1-liner changes that change `BaseTest` to `GATKBaseTest`, if such changes are necessary (i.e. if not already done). 
Thanks!